template2.html

```diff
@@ -0,0 +1,180 @@
+            </div>
+            <footer>
+               <div id="skills_3" class="skills">
+                 <p>html</p>
+                 <p>css</p>
+                 <p>javaScript</p>
+               </div>
+               <div class="external_link">
+                   <a href="#">See Project</a>
+               </div>
+              </footer>
+  
+            
+          </section>
+          <section id="about_me">
+              <h1>About<br> Myself</h1>
+              <p id="presentation">Hello I’m a software developer! I can help you build a product , 
+                  feature or website Look through some of my work and experience!
+                   If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+              <div id="profile_link">
+                <h2>LET’S CONNECT</h2>
+                <ul>
+                    <li><a href="#"><img src="statics/images/github.png" alt="github profilelink"></a></li>
+                    <li><a href="#"><img src="statics/images/Linkedin_icon.png" alt="linkedin profile link"></a></li>
+                    <li><a href="#"><img src="statics/images/doits.png" alt="doit"></a></li>
+                    <li><a href="#"><img src="statics/images/twitter.png" alt="twitter profile link"></a></li>
+                    <li><a href="#"><img src="statics/images/M.png" alt="M link"></a></li>
+                </ul>
+                <a id="my_resume" href="#">Get my resume</a> 
+              </div>
+              <div id="my_skills">
+                  <div id="languages_skills"> 
+                      <div id="spec_name">
+                          <p>Langguages</p>
+                          <div><img src="statics/images/fleche_dessous.png" alt="selected"></div>
+                      </div>
+                      <ul id="programmes">
+                        <li class="pro"> 
+                            <img src="statics/images/java.png" alt="javascript image"> 
+                            <p>JavaSript</p>
+                        </li>
+                        <li class="pro">
+                            <img src="statics/images/html.png" alt="html image">
+                            <p>HTML</p>
+                        </li>
+                        <li class="pro">
+                            <img src="statics/images/css.png" alt="css image">
+                            <p>CSS</p>
+                        </li>
+                      </ul>
+                  </div>
+                  <div id="frameworks">
+                      <p>Frameworks</p>
+                      <div><img src="statics/images/select.png" alt="select"></div>
+                  </div>
+                  <div id="l1"> <hr/></div>
+                  <div id="sk">
+                    <p>Skills</p>
+                    <div><img src="statics/images/select.png" alt="select"></div>
+                  </div>
+                  <div class="ligne" id="l2"> <hr/></div>
+                </div>
```

Comment:
- [ ] Languages, frameworks, and skills should also be `li` tags inside a `ul` tag, please consider changing this. This way your code will be more semantic.